lib/form/html.js

```diff
@@ -107,35 +106,18 @@ const addRequired = (html, input, index) => {
         return;
       }
 
-      // regex to get first class attributes from the captured label
-      const regexClass = new RegExp(`.*(${stringClass})`);
-
-      // regex to get classes out of `class`
-      const regexClasses = new RegExp('class=["\']([\\w\\s\\W]+?)["\']');
-
       // regex to search for `for`
       const reFor = new RegExp(`(${stringFor})`);
-
-      // first, check if `class` exists in label
-      if (Array.isArray(label[1].match(regexClass))) {
-        const cls = label[1].match(regexClass);
-        const classAttr = cls[1].match(regexClasses);
-        const classes = classAttr[1].split(/\s+/).concat([level]);
-        const newClass = `class="${classes.join(' ')}"`;
-        render = render.replace(label[0], label[0].replace(classAttr[0], newClass));
-      }
-      else {
-        // else, we add class after the `for` attribute
-        render = render.replace(reFor, `for="${inp.id}" class="${level}"`);
-      }
+      render = render.replace(reFor, `for="${inp.id}"`);
 
       // add required to input
       const reName = new RegExp(`name=['"]\\s*${inp.name}\\s*['"]`);
       render = render.replace(reName, `name="${inp.name}" aria-required="true" required`);
 
       // Add required text to label
       const reLabel = new RegExp('</label>');
-      render = render.replace(reLabel, `<mark class="${level}">required to ${required}</mark></label>`);
+      render = render.replace(reLabel, `\u00a0<mark class="${level}">required to ${required}</mark>`);
```

Comment:
Yes. I couldn't get it to work with just a space, but the unicode space worked. @Snugug may have a better way to do it, I just couldn't figure it out.